### PR TITLE
API docs migration 5

### PR DESCRIPTION
**Pull request by AaronForinton**
- web.dev/cors-rfc1918-feedback -> https://developer.chrome.com/articles/cors-rfc1918-feedback/
- web.dev/file-handling -> https://developer.chrome.com/articles/file-handling
- web.dev/hid -> https://developer.chrome.com/articles/hid
- web.dev/isinputpending -> https://developer.chrome.com/articles/isinputpending
- web.dev/webcodecs -> https://developer.chrome.com/articles/webcodecs

### CODE REVIEW

**Review by Alicegao88 — COMMENTED** [review]
``

**Comment by AaronForinton** [comment]
The build errors on this are for brighter minds than mine, I'm afraid @rachelandrew 

https://github.com/GoogleChrome/developer.chrome.com/actions/runs/3451953688/jobs/5761526701

File: site/en/articles/file-handling/index.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+<figure>
+  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/TMh8Qev0XdwgIx7jJlP5.png", alt="The macOS finder window with an Excalidraw file.", width="800", height="422" %}
+  <figcaption>
+     Double click or right click a file in your operating system's file explorer.
+  </figcaption>
+</figure>
+
+<figure>
+  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt="The context menu that appears when right clicking a file with the 'Open with… Excalidraw' item highlighted.", width="488", height="266" %}
```
Comment (rachelandrew) [inline site/en/articles/file-handling/index.md:184]:
```suggestion
  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt="The context menu that appears when right clicking a file with the Open with… Excalidraw' item highlighted.", width="488", height="266" %}
```

File: site/en/articles/file-handling/index.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+
+<figure>
+  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/TMh8Qev0XdwgIx7jJlP5.png", alt="The macOS finder window with an Excalidraw file.", width="800", height="422" %}
+  <figcaption>
+     Double click or right click a file in your operating system's file explorer.
+  </figcaption>
+</figure>
+
+<figure>
+  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt="The context menu that appears when right clicking a file with the Open with… Excalidraw' item highlighted.", width="488", height="266" %}
```
Comment (rachelandrew) [inline site/en/articles/file-handling/index.md:184]:
```suggestion
  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt="The context menu that appears when right clicking a file with the Open with… Excalidraw item highlighted.", width="488", height="266" %}
```

File: site/pt/articles/file-handling/index.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+<figure>{% Img src = "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/TMh8Qev0XdwgIx7jJlP5.png", alt = "A janela do localizador do macOS com um arquivo Excalidraw.", width = "800", height = "422" %}<figcaption> Clique duas vezes ou clique com o botão direito em um arquivo no explorador de arquivos do seu sistema operacional.</figcaption></figure>
+
+<figure>{% Img src = "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt = "O menu de contexto que aparece ao clicar com o botão direito em um arquivo com o item 'Abrir com… Excalidraw'.", width = "488", height = " 266 " %}<figcaption> Excalidraw é o gerenciador de arquivos padrão para arquivos <code>.excalidraw</code></figcaption>.</figure>
```
Comment (rachelandrew) [inline site/pt/articles/file-handling/index.md:142]:
```suggestion
<figure>{% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/TMh8Qev0XdwgIx7jJlP5.png", alt="A janela do localizador do macOS com um arquivo Excalidraw.", width="800", height="422" %}<figcaption> Clique duas vezes ou clique com o botão direito em um arquivo no explorador de arquivos do seu sistema operacional.</figcaption></figure>

<figure>{% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/wCNbMl6kJ11XziG3LO65.png", alt="O menu de contexto que aparece ao clicar com o botão direito em um arquivo com o item 'Abrir com… Excalidraw'.", width="488", height=" 266 " %}<figcaption> Excalidraw é o gerenciador de arquivos padrão para arquivos <code>.excalidraw</code></figcaption>.</figure>
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**